bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/BaseThingHandler.java

```diff
@@ -552,33 +514,22 @@ protected boolean isLinked(String channelId) {
     }
 
     /**
-     * This method is called, when the according {@link ThingHandler} of the
-     * bridge was initialized. If the thing of this handler does not have a
-     * bridge, this method is never called. This method can be overridden by
-     * subclasses.
-     *
-     * @param thingHandler
-     *            thing handler of the bridge
-     * @param bridge
-     *            bridge
+     * Returns whether the thing has already been initialized.
+     * 
+     * @return true if thing is initialized, false otherwise
      */
-    protected void bridgeHandlerInitialized(ThingHandler thingHandler, Bridge bridge) {
-        // can be overridden by subclasses
+    protected boolean thingIsInitialized() {
+        return getThing().getStatus() == ThingStatus.ONLINE || getThing().getStatus() == ThingStatus.OFFLINE;
+    }
+    
+    @Override
+    public void bridgeHandlerInitialized(ThingHandler thingHandler, Bridge bridge) {
```

Comment:
what was the reason to change this method to public? This breaks existing code, since this will now see errors like:
```
Cannot reduce the visibility of the inherited method from BaseThingHandler	RFXComHandler.java
```